deepchem/models/__init__.py

```diff
@@ -2,25 +2,28 @@
 Gathers all models in one place for convenient imports
 """
 # flake8: noqa
-from deepchem.models.models import Model
-from deepchem.models.keras_model import KerasModel
-from deepchem.models.multitask import SingletaskToMultitask
-from deepchem.models.callbacks import ValidationCallback
-
-from deepchem.models.IRV import MultitaskIRVClassifier
-from deepchem.models.robust_multitask import RobustMultitaskClassifier
-from deepchem.models.robust_multitask import RobustMultitaskRegressor
-from deepchem.models.progressive_multitask import ProgressiveMultitaskRegressor, ProgressiveMultitaskClassifier
-from deepchem.models.graph_models import WeaveModel, DTNNModel, DAGModel, GraphConvModel, MPNNModel
-from deepchem.models.scscore import ScScoreModel
-
-from deepchem.models.seqtoseq import SeqToSeq
-from deepchem.models.gan import GAN, WGAN
-from deepchem.models.molgan import BasicMolGANModel
-from deepchem.models.cnn import CNN
-from deepchem.models.text_cnn import TextCNNModel
-from deepchem.models.atomic_conv import AtomicConvModel
-from deepchem.models.chemnet_models import Smiles2Vec, ChemCeption
+try:
+  from deepchem.models.models import Model
```

Comment:
Model doesn't use TensorFlow.  It's the abstract parent class of all models.  If we can't import it, we can't import *any* model.

Reply:
Fixed it